digest.cfg

```diff
@@ -18,8 +18,15 @@ medium_access_token:
 output_file_name_pattern: {author}_{msid:0>5}.docx
 # DOI formatting
 doi_pattern: 
+# IIIF image server settings
+iiif_image_uri:
+iiif_image_source_uri:
+iiif_info_url: 
 
 [elife]
 medium_license: cc-40-by
 medium_image_url: https://iiif.elifesciences.org/digests:{msid:0>5}%2F{file_name}/full/full/0/default.jpg
 doi_pattern: https://doi.org/10.7554/eLife.{msid:0>5}
+iiif_image_uri: https://iiif.elifesciences.org/digests:{msid:0>5}%2F{file_name}
+iiif_image_source_uri: https://iiif.elifesciences.org/digests:{msid:0>5}%2F{file_name}/full/full/0/default.jpg
```

Comment:
just a heads up that this will always be using the full resolution image, it works although it can be heavy